.circleci/config.yml

```diff
@@ -75,7 +75,7 @@ jobs:
         - run: apt-get install -y sudo
         - run: sudo apt-get install -y tcl
         - run: sudo apt-get install -y expect
-        - run: sudo apt-get install -y zip
+        - run: sudo apt-get update && sudo apt-get install -y zip
```

Comment:
May be move `sudo apt-get update` to the line 75 as a separate command. This will ensure all the subsequent `apt-get install *` will work